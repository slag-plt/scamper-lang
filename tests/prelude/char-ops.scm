(digit-value #\5)
(digit-value #\0)

(char->integer #\a)
(integer->char 97)

(char-upcase #\a)
(char-upcase #\A)
(char-downcase #\a)
(char-downcase #\A)
(char-foldcase #\a)
(char-foldcase #\A)