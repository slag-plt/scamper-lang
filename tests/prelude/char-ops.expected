5
0
97
#\a
#\A
#\A
#\a
#\a
#\a
#\a
